compiler/il/OMRNode.cpp

```diff
@@ -2415,26 +2415,84 @@ OMR::Node::computeIsInternalPointer()
  */
 bool
 OMR::Node::computeIsCollectedReference()
+   {
+   TR::Compilation * comp = TR::comp();
+   TR::NodeChecklist processedNodesCollected(comp);
+   TR::NodeChecklist processedNodesNotCollected(comp);
+   return (self()->computeIsCollectedReferenceImpl(true, processedNodesCollected, processedNodesNotCollected) != TR_no);
+   }
+
+static void
+recordProcessedNodeResult(TR::Node *node, TR_YesNoMaybe collectedness, TR::NodeChecklist &processedNodesCollected, TR::NodeChecklist &processedNodesNotCollected)
```

Comment:
could I suggest that this method return the collectedness so that in the cases where the result type is hardcoded you only need to do `return recordProcessedNodeResult(…)` which prevents duplicate editing errors where the result has to change?

Reply:
Addressed.